cmd/root.go

```diff
@@ -1,41 +0,0 @@
-package cmd
```

Comment:
the original layout is meant to follow [`golang-standards` project layout](https://github.com/golang-standards/project-layout) which is also used by some [major libraries](https://github.com/golang-standards/project-layout/tree/master/cmd).

I think it'll be better to follow that for now.

Reply:
Do you think we need to synchronize between the boilerplate and the one mentioned in onboarding? If so, which one should we choose?
https://github.com/KurioApp/onboarding/tree/master/Go

Reply:
ah, I missed that onboarding.
but moving forward, I propose to follow the more adopted standard. so yeah, we may have to update the onboarding file.